Utilities/wearable-packager/marketplace-packager.js

```diff
@@ -0,0 +1,144 @@
+
+    var tablet = Tablet.getTablet('com.highfidelity.interface.tablet.system');    
+
+    function handleMousePress(entityID) {
+        if (prevID !== entityID) {
+            Selection.addToSelectedItemsList(listName, listType, entityID);
+            prevID = entityID;
+        }
+        tablet.emitScriptEvent(entityID);
+    }
+
+    function handleMouseLeave(entityID) {
+        if (prevID !== 0) {
+            Selection.removeFromSelectedItemsList("contextOverlayHighlightList", listType, prevID);
+            prevID = 0;
+        }
+    }
+
+    var baseUserdata = {
+        "Attachment": {
+            "action": "attach",
+            "joint": "Hips",
+            "attached" : false,
+            "options": {
+                "translation": {
+                    "x": 0,
+                    "y": 0,
+                    "z": 0
+                },
+                "scale": 1
+            }
+        },
+        "grabbableKey": {
+            "cloneable": false,
+            "grabbable": true
+        }
+    };
+
+    var exportProperties = {
+        type: "Model",
+        clientOnly: 1,
+        parentID: "{00000000-0000-0000-0000-000000000001}",
+        owningAvatarID: "{00000000-0000-0000-0000-000000000000}",
+        visible: 1,
+        shapeType: "box",
+        collidesWith: "",
+        collisionMask: 0
+    };
+
+    var button = tablet.addButton({
+        text: APP_NAME,
+        icon: APP_ICON
+    });
+
+    
+    function maybeExited() {
+        Entities.clickReleaseOnEntity.disconnect(handleMousePress);
+        Entities.hoverLeaveEntity.disconnect(handleMouseLeave);
+        tablet.screenChanged.disconnect(maybeExited);
+    }
+
+    function clicked(){
+        tablet.gotoWebScreen(APP_URL);
+        Entities.clickReleaseOnEntity.connect(handleMousePress);
+        Entities.hoverLeaveEntity.connect(handleMouseLeave);
+        Script.setTimeout(function(){
+            tablet.screenChanged.connect(maybeExited); 
+        }, 2000);
+    }
+    button.clicked.connect(clicked);
+
+    function onFileSaveChanged(filename){
+        Window.saveFileChanged.disconnect(onFileSaveChanged);
+        if (filename !== "") {
+            var success = Clipboard.exportEntities(filename, [entityIDToExport]);
+            if (!success) {
+                // No luck, failed
+            }
+        }
+        Entities.deleteEntity(entityIDToExport);                        
+    }
+
+    function onWebEventReceived(event){
+        if (typeof(event) === "string") {
+            event = JSON.parse(event);
+        }
+        if (event.type === "submit") {
+            var entityID = event.entityID;
+            var joint = event.joint;
+
+            var newExportProperties = exportProperties;
+            var newUserDataProperties = baseUserdata;
+    
+            var properties = Entities.getEntityProperties(entityID, ['modelURL', 'dimensions', 'script']);
+
+            newUserDataProperties["Attachment"].joint = joint;
```

Comment:
I'd prefer to change `newUserDataProperties["Attachment"].joint` -> `newUserDataProperties.Attachment.joint`, although the code will still fail when there is no Attachment key in the property in both cases. Try safe-guarding that case.